lib/v2/china/router.js

```diff
@@ -1,4 +1,5 @@
 module.exports = (router) => {
     router.get('/news/military', require('./news/military/news.js'));
     router.get('/news/:category?', require('./news/highlights/news.js'));
+    router.get('/finance/:category?/:limit?', require('./finance/finance.js'));
```

Comment:
```suggestion
    router.get('/finance/:category?/:limit?', require('./finance/finance.js'));
    router.get('/news/military', require('./news/military/news.js'));
    router.get('/news/:category?', require('./news/highlights/news.js'));
```
Please insert new entries in alphabetical order.